README.rst

```diff
@@ -6,62 +6,81 @@ Astropy
 
     |Actions Status| |CircleCI Status| |Coverage Status| |PyPI Status| |Documentation Status| |Pre-Commit| |Ruff| |Zenodo|
 
-The Astropy Project (http://astropy.org/) is a community effort to develop a
-single core package for Astronomy in Python and foster interoperability between
-Python astronomy packages. This repository contains the core package which is
-intended to contain much of the core functionality and some common tools needed
-for performing astronomy and astrophysics with Python.
+The Astropy Project (https://astropy.org/) is a community effort to develop a single core package for Astronomy in Python and foster interoperability between Python astronomy packages. This repository contains the core package which is intended to contain much of the core functionality and some common tools needed for performing astronomy and astrophysics with Python.
 
-Releases are `registered on PyPI <https://pypi.org/project/astropy>`_,
-and development is occurring at the
-`project's GitHub page <http://github.com/astropy/astropy>`_.
+Table of Contents
+=================
```

Comment:
Our README is short enough that we don't benefit from having a table of contents.

Reply:
I kinda like it. It tells me at a glance what is in the README.